### PR TITLE
am adaugat bbr, tabel si adresa

### PR DESCRIPTION
**Pull request by florin-bizgan**

### CODE REVIEW

**Review by MariaTeo — APPROVED** [review]
nice

**Review by marianzburlea — CHANGES_REQUESTED** [review]
update indenting

File: src/007florin/form/index.pug [hunk 1]
```diff
@@ -1,21 +1,122 @@
...
+      type="text"
+      id="last-name"
+      value=""
+    )
+    br
+    label(for="birth-date") Day of birth
+    br
+    input(
+      type="date"
+      name=""
```
Comment (marianzburlea) [inline src/007florin/form/index.pug:37]: id="birth-date"

**Review by marianzburlea — CHANGES_REQUESTED** [review]
update indenting

**Pull request merged**